### PR TITLE
Update ZFS arc min/max sysctl loader variables

### PR DESCRIPTION
**Pull request by xor-gate**
Update arc min and max loader syctl variable names based on 13.1-RELEASE-p2

### CODE REVIEW

File: documentation/content/en/books/handbook/zfs/_index.adoc [hunk 1]
```diff
@@ -2619,9 +2619,9 @@ If a user has the `snapshot` permission and the `allow` permission, that user ca
 
 Adjust tunables to make ZFS perform best for different workloads.
 
-* [[zfs-advanced-tuning-arc_max]] `_vfs.zfs.arc_max_` - Upper size of the <<zfs-term-arc,ARC>>. The default is all RAM but 1 GB, or 5/8 of all RAM, whichever is more. Use a lower value if the system runs any other daemons or processes that may require memory. Adjust this value at runtime with man:sysctl[8] and set it in [.filename]#/boot/loader.conf# or [.filename]#/etc/sysctl.conf#.
+* [[zfs-advanced-tuning-arc_max]] `_vfs.zfs.arc.max_` (legacy `_vfs.zfs.arc_max`) - Upper size of the <<zfs-term-arc,ARC>>. The default is all RAM but 1 GB, or 5/8 of all RAM, whichever is more. Use a lower value if the system runs any other daemons or processes that may require memory. Adjust this value at runtime with man:sysctl[8] and set it in [.filename]#/boot/loader.conf# or [.filename]#/etc/sysctl.conf#.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/zfs/_index.adoc:2622]: I'd be explicit about versions and use:
> `_vfs.zfs.arc.max_` starting with 13.x (`_vfs.zfs.arc_max` for 12.x)

File: documentation/content/en/books/handbook/zfs/_index.adoc [hunk 2]
```diff
@@ -2619,9 +2619,9 @@ If a user has the `snapshot` permission and the `allow` permission, that user ca
...
-* [[zfs-advanced-tuning-arc_min]] `_vfs.zfs.arc_min_` - Lower size of the <<zfs-term-arc,ARC>>. The default is one half of `vfs.zfs.arc_meta_limit`. Adjust this value to prevent other applications from pressuring out the entire <<zfs-term-arc,ARC>>. Adjust this value at runtime with man:sysctl[8] and in [.filename]#/boot/loader.conf# or [.filename]#/etc/sysctl.conf#.
+* [[zfs-advanced-tuning-arc_min]] `_vfs.zfs.arc.min_` (legacy `_vfs.zfs.arc_min`) - Lower size of the <<zfs-term-arc,ARC>>. The default is one half of `vfs.zfs.arc_meta_limit`. Adjust this value to prevent other applications from pressuring out the entire <<zfs-term-arc,ARC>>. Adjust this value at runtime with man:sysctl[8] and in [.filename]#/boot/loader.conf# or [.filename]#/etc/sysctl.conf#.
```
Comment (pauamma) [inline documentation/content/en/books/handbook/zfs/_index.adoc:2624]: Same here.

**Review by pauamma — CHANGES_REQUESTED** [review]

**Comment by xor-gate** [comment]
Good comments @pauamma fixed in https://github.com/freebsd/freebsd-doc/pull/92/commits/99cc596880c90404afea83ee75ffb52e8899926f. I did not dig deep to get to know which when was introduced and deprecated. Thanks!

**Review by pauamma — COMMENTED** [review]
Text LGTM now. Will try building as soon as I remember how to get a raw diff from GitHub.

**Comment by pauamma** [comment]
Almost there. vfs.zfs.arc_meta_limit became vfs.zfs.arc.meta_limit in 13.x as well. (Didn't notice on first round.)

How do you want to credited for this patch? (Name and email address.)

**Comment by xor-gate** [comment]
Did a vim search on leftovers but found some more inconsistencies. You can credit me on Jerry Jacobs<jerry.jacobs[_at_]xor-gate.org>. Thanks for pointing out the meta_limit. I did not a scan of legacy on the whole sysctl range.

**Comment by xor-gate** [comment]
Review comments again fixed in https://github.com/freebsd/freebsd-doc/pull/92/commits/6d509de41e5082f056d00fff8efcf1f36f16a461

**Comment by pauamma** [comment]
Assumption check: do you want your email address recorded as-is with the obfuscation?

**Review by pauamma — APPROVED** [review]

**Comment by xor-gate** [comment]
> Assumption check: do you want your email address recorded as-is with the obfuscation?

Remove the obfuscation, replace with `@` :-). Email harvest bots can be nasty 👍 . Thanks for having me fixing this issues in the docs, it was nice working together to fix this @pauamma

**Comment by pauamma** [comment]
> Thanks for having me fixing this issues in the docs, it was nice working together to fix this @pauamma

Thank you for submitting it! Committed.

**Pull request closed without merging**